docs/_interface/Private-Network.md

```diff
@@ -3,103 +3,84 @@ title: Private Networks
 sort_key: B
 ---
 
-This guide explains how to set up a private network of multiple Geth nodes. An Ethereum
-network is a private network if the nodes are not connected to the main network. In this
-context private only means reserved or isolated, rather than protected or secure.
+This guide explains how to set up a private network of multiple Geth nodes. An Ethereum network is private if the nodes are not connected to the main network. In this
+context private only means reserved or isolated, rather than protected or secure. A fully controlled, private Ethereum network is useful as a backend for network integration testing for core developers working on issues realting to networking/blockchain syncing etc. Private networks are also useful for Dapp developers testing multi-block and multi-user scenarios. 
```

Comment:
`A fully controlled, private...`

this sentence is too long.

Reply:
ok, shortened.